actions/workflows/promote_package.yaml

```diff
@@ -0,0 +1,100 @@
+---
+version: '2.0'
+
+st2cd.promote_package:
+    description: Promote a package to production.
+    type: direct
+    input:
+      - package
+      - version
+      - revision
+      - os
+      - repo
+    tasks:
+
+      init:
+        action: core.local
+        input:
+          cmd: "mkdir -p /tmp/st2-up/{ubuntu/trusty,debian/wheezy,debian/jessie,el/6,el/7}"
```

Comment:
Would inject some unique reference like revision etc in the directory name. This way different instances of this workflow will not clash against each other.